datanode/sqlstore/migrations/0063_ledger_types.sql

```diff
@@ -0,0 +1,39 @@
+-- +goose Up
+CREATE TYPE ledger_movement_type AS ENUM (
```

Comment:
Is there are particular reason for changing this to an enum? I'm asking because its a little bit less error prone when its not an enum, as we often have situations where a new one is added to the protos but not added to the schema and it causes the data-node to panic.

Reply:
Since we are usually using enum for this kind of field, I made that part consistent as part of the code cleanup. I don't think we are earning anything regarding performance or anything, except getting a strong data model, so I am happy to revert that part if we are prone to panic.